cmd/support.go

```diff
@@ -1,18 +1,102 @@
 package cmd
 
 import (
+	"errors"
 	"fmt"
+	"net/url"
+	"regexp"
 
+	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
 )
 
+const (
+	baseURL              = "https://connect.redhat.com/support/technology-partner/#/case/new?"
+	typeParam            = "type"
+	typeValue            = "CERT"
+	sourceParam          = "source"
+	sourceValue          = "preflight"
+	certProjectTypeParam = "cert_project_type"
+	certProjectIDParam   = "cert_project_id"
+	pullRequestURLParam  = "pull_request_url"
+)
+
 var supportCmd = &cobra.Command{
 	Use:   "support",
 	Short: "Submits a support request",
-	Long: `This command will submit a support request to Red Hat along with the logs from the latest Preflight checck.
+	Long: `This interactive command will generate a URL; based on user input which can then be used to create a Red Hat Support Ticket.
 	This command can be used when you'd like assistance from Red Hat Support when attempting to pass your certification checks. `,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Support command not implemented")
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		certProjectTypeLabel := promptui.Select{
+			Label: "Select a Certification Project Type",
+			Items: []string{"Container Image", "Operator Bundle Image"},
+		}
+
+		_, certProjectTypeValue, err := certProjectTypeLabel.Run()
+
+		if err != nil {
+			fmt.Println("Prompt Failed, Please Try re-running support command.")
+			return err
+		}
+
+		fmt.Printf("You Selected:  %q\n", certProjectTypeValue)
+
+		certProjectIDLabel := promptui.Prompt{
+			Label: "Please Enter Connect Certification Project ID",
+			Validate: func(s string) error {
+				isLegacy, _ := regexp.MatchString(`^p.*`, s)
```

Comment:
Could we just strip these known prefixes when we identify them? Maybe log a debug or trace message indicating that it was stripped and try the result?

Reply:
I initially thought about that, but went with this blocking approach since it is built into the prompt library and I feel that gives a better user experience, since we are going with an interactive approach for this.